### PR TITLE
Virt Blocks update

### PR DESCRIPTION
**Pull request by andreabolognani**
Build the Rust code along with the Go code; in addition to building, run the test suite and the example programs too.

Doing this requires using a custom container image that includes a Rust compiler.

### CODE REVIEW

File: github/ci/prow/files/jobs/virtblocks/virtblocks/virtblocks-periodics.yaml [hunk 1]
```diff
@@ -5,13 +5,39 @@ presubmits:
     always_run: true
     spec:
       containers:
-        - image: kubevirt/golang-libtool:1.12.5-r1
+        - image: quay.io/virblocks/buildenv:1.0.0
```
Comment (mfranczy) [inline github/ci/prow/files/jobs/virtblocks/virtblocks/virtblocks-periodics.yaml]: should not the name be `virtblocks`?

Comment (andreabolognani) [reply github/ci/prow/files/jobs/virtblocks/virtblocks/virtblocks-periodics.yaml]: > should not the name be `virtblocks`?

It definitely should, good catch! I'll fix it right away.

Comment (andreabolognani) [reply github/ci/prow/files/jobs/virtblocks/virtblocks/virtblocks-periodics.yaml]: Done!

**Pull request merged**